src/EnergyPlus/Pumps.cc

```diff
@@ -2041,8 +2042,7 @@ void CalcPumps(EnergyPlusData &state, int const PumpNum, Real64 const FlowReques
     //****************************!
     //***** CALCULATE POWER (1) **!
     //****************************!
-    if (PumpType == iPumpType::ConSpeed || PumpType == iPumpType::VarSpeed || PumpType == iPumpType::Cond) {
-
+    if (BITF_TEST_ANY(BITF(PumpType), BITF(iPumpType::ConSpeed) | BITF(iPumpType::VarSpeed) | BITF(iPumpType::VarSpeed))) {
         VolFlowRate = state.dataPumps->PumpMassFlowRate / LoopDensity;
```

Comment:
The last one should be iPumpType::Cond, no?

Reply:
Yes, thanks for noticing this.